docs/docs/config/system.md

```diff
@@ -22,9 +22,10 @@ Definition file: [zmk/app/Kconfig](https://github.com/zmkfirmware/zmk/blob/main/
 
 ### HID
 
-| Config                                | Type | Description                                       | Default |
-| ------------------------------------- | ---- | ------------------------------------------------- | ------- |
-| `CONFIG_ZMK_HID_CONSUMER_REPORT_SIZE` | int  | Number of consumer keys simultaneously reportable | 6       |
+| Config                                | Type | Description                                                    | Default |
+| ------------------------------------- | ---- | -------------------------------------------------------------- | ------- |
+| `CONFIG_ZMK_HID_INDICATORS`           | bool | Enable reciept of HID/LED indicator state from connected hosts | n       |
```

Comment:
```suggestion
| `CONFIG_ZMK_HID_INDICATORS`           | bool | Enable receipt of HID/LED indicator state from connected hosts | n       |
```
Is there a reason not to default this to on?

Reply:
After the USB BOOT word caused some unexpected issues (for you even), I'm wary to enable this by default until the actual version merged into ZMK `main` gets a bit more extensive testing by folks.